src/providers/swagger/swagger.provider.ts

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
here's the file where we parse and transform the endpoints in proper swagger configuration